src/lib/features/client-feature-toggles/delta/delta-cache.ts

```diff
@@ -12,6 +12,22 @@ export class DeltaCache {
     constructor(hydrationEvent: DeltaHydrationEvent, maxLength: number = 20) {
         this.hydrationEvent = hydrationEvent;
         this.maxLength = maxLength;
+
+        this.addBaseEventFromHydration(hydrationEvent);
```

Comment:
This is important change. Previously when we were initializing environment, we just filled hydration event. But hydration event changes in time, so we also need to add an event with same revision id as base event to event queue.

This way if customer got hydration with rev 100, and hydration now moves to 101, customer is coming back with 100, we need to have reference to 100.